UnitTests/GitCommandsTests/GitModuleTest.cs

```diff
@@ -452,5 +453,60 @@ public void AddRemote()
             Assert.AreEqual("Please enter a name.", _gitModule.AddRemote("", path));
             Assert.AreEqual("Please enter a name.", _gitModule.AddRemote(null, path));
         }
+
+        [Test]
+        public void GetSubmodulesLocalPaths()
+        {
+            var moduleTestHelpers = new List<CommonTestUtils.GitModuleTestHelper>();
+            try
+            {
+                const int numModules = 4;
+
+                for (int i = 0; i < numModules; ++i)
+                {
+                    moduleTestHelpers.Add(new CommonTestUtils.GitModuleTestHelper($"repo{i}"));
+                }
+
+                foreach (var helper in moduleTestHelpers)
+                {
+                    // TODO: move to GitModuleTestHelper constructor (can't assume all users have global user.name/email configured)
```

Comment:
My current setup is to have no global user.name/email configured because I work on different repos where I use personal or work identity, so to make sure I don't confuse them, I always set them locally. I think we should add these lines to the GitModuleTestHelper constructor so this will work in an environment like mine.

Reply:
https://git-scm.com/docs/git-config#_includes  You can store work repos in one folder and and personal in another.  In your global gitconfig include work or personal git config for which user and such.

Reply:
That's good to know, but I still think we should add the local config for user/email in case users don't do this. Do you agree?

Reply:
Sure. Just pointing out an easier way to manage your git config.

Reply:
Sure, I have no problems with such change